pkg/autoupdate/autoupdate_test.go

```diff
@@ -18,7 +18,8 @@ func TestCreateTUFRepoDirectory(t *testing.T) {
 	localTUFRepoPath, err := ioutil.TempDir("", "")
 	require.NoError(t, err)
 
-	require.Nil(t, createTUFRepoDirectory(localTUFRepoPath, "pkg/autoupdate/assets", AssetDir))
+	u := &Updater{}
+	require.Nil(t, u.createTUFRepoDirectory(localTUFRepoPath, "pkg/autoupdate/assets", AssetDir))
```

Comment:
nit: `require.NoError(...)` is slightly more expressive here, but this is fine.

Reply:
Not my code. But that's reasonable, I fixed it